src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -114,6 +115,8 @@ public void snapshots(UIThreadSet<Activity> liveActivities, OutputStream out) th
                 Log.e(LOGTAG, "Exception thrown during screenshot attempt", e);
             }
         }
+        writer.write("\"\"");
+        writer.flush();
```

Comment:
maybe move it into the `catch (final TimeoutException e)` block to make it more specific?